sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
To elaborate: one might conclude that srepr of a float should prints its `_mpf`_`; but this is not so readable.  We want a similar guarantee of `eval(srepr(...))` being identity map.